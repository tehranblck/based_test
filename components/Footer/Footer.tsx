import Link from "next/link";
import { FaFacebook, FaInstagram, FaTelegram, FaWhatsapp, FaYoutube } from "react-icons/fa";
import { IoLogoWhatsapp } from "react-icons/io5";

export default function Footer() {
    const currentYear = new Date().getFullYear();

    const contactLinks = [
        { text: "Əlaqə nömrəsi: +994775450755", href: "tel:+994775450755" },
        { text: "Email: contact@based.az", href: "mailto:contact@based.az" }
    ];

    const discoverLinks = [
        "Hesablar və Abunəliklər",
        "Proqram Lisenziyaları",
        "Panel Xidmətləri və Alətlər",
        "Sayt və Marketing Halləri",
        "Rəqəmsal Məhsullar və Alətlər"
    ];

    const advantagesList = [
        "Sürətli Təhvil",
        "Etibarlı Xidmət",
        "24/7 Dəstək",
        "Sərfəli Qiymətlər",
        "Avtomatlaşdırılmış Sistem"
    ];

    const socialLinks = [
        { icon: FaWhatsapp, href: "https://wa.me/994775450755", label: "WhatsApp", color: "hover:bg-green-500" },
        { icon: FaInstagram, href: "https://instagram.com/based.az", label: "Instagram", color: "hover:bg-pink-500" },
        { icon: FaTelegram, href: "https://t.me/basedaz", label: "Telegram", color: "hover:bg-blue-500" },
        { icon: FaFacebook, href: "https://facebook.com/based.az", label: "Facebook", color: "hover:bg-blue-600" },
        { icon: FaYoutube, href: "https://youtube.com/@based.az", label: "YouTube", color: "hover:bg-red-500" }
    ];

    return (
        <>
            <footer style={{ borderTopLeftRadius: "20px", borderTopRightRadius: "20px" }} className="bg-card/60 text-foreground  relative">
                <div className=" max-w-[90rem] mx-auto px-4 py-12">
                    <div className="mb-10 rounded-xl border border-border bg-primary/5 p-4 sm:p-6 flex flex-col md:flex-row items-center justify-between gap-3">
                        <div className="text-center md:text-left">
                            <p className="text-sm text-muted-foreground">Sualınız var? Dəstək komandamız 24/7 onlayndır.</p>
                            <p className="text-base font-semibold text-secondary">Bizə yazın, dərhal cavab alacaqsınız.</p>
                        </div>
                        <div className="flex items-center gap-3">
                            <Link href="https://wa.me/994775450755" target="_blank" rel="noopener noreferrer" className="inline-flex items-center justify-center rounded-md bg-secondary text-secondary-foreground hover:bg-secondary/90 font-medium px-4 py-2 transition-colors" aria-label="WhatsApp ilə əlaqə">WhatsApp</Link>
                            <Link href="https://t.me/basedaz" target="_blank" rel="noopener noreferrer" className="inline-flex items-center justify-center rounded-md bg-muted text-foreground hover:bg-accent font-medium px-4 py-2 transition-colors" aria-label="Telegram ilə əlaqə">Telegram</Link>
                        </div>
                    </div>
                    <div className="grid grid-cols-1 md:grid-cols-4 gap-8">
                        {/* Company Info Section */}
                        <div className="space-y-4">
                            <div className="flex items-center space-x-3">
                                <span className="text-2xl font-bold text-secondary">BASED.AZ</span>
                            </div>
                            <p className="text-muted-foreground leading-relaxed text-sm">
                                Rəqəmsal hesablar, abunəliklər, oyunlar, proqram lisenziyaları və digər bütün ehtiyaclarınızı tək platformada rahat, sürətli və sərfəli şəkildə qarşılamaq mümkündür. Seçim edin, arxayın olun və xidmət keyfiyyətini hiss edin.
                            </p>
                        </div>

                        {/* Contact Section */}
                        <div className="space-y-4">
                            <h3 className="text-foreground font-semibold text-lg">Əlaqə</h3>
                            <div className="space-y-3">
                                {contactLinks.map((contact, index) => (
                                    <div key={index}>
                                        <Link
                                            href={contact.href}
                                            className="text-muted-foreground hover:text-foreground transition-colors duration-200 text-sm"
                                        >
                                            {contact.text}
                                        </Link>
                                    </div>
                                ))}
                            </div>

                            <div className="flex space-x-3 pt-4">
                                {socialLinks.map((social, index) => (
                                    <Link
                                        key={index}
                                        href={social.href}
                                        target="_blank"
                                        rel="noopener noreferrer"
                                        className={`w-8 h-8 bg-muted hover:bg-secondary rounded flex items-center justify-center transition-all duration-200 group`}
                                        aria-label={social.label}
                                    >
                                        <social.icon className="w-4 h-4 text-muted-foreground group-hover:text-secondary-foreground" />
                                    </Link>
                                ))}
                            </div>
                        </div>

                        {/* Discover Section */}
                        <div className="space-y-4">
                            <h3 className="text-foreground font-semibold text-lg">Kəşf et</h3>
                            <ul className="space-y-2">
                                {discoverLinks.map((link, index) => (
                                    <li key={index} className="flex items-center">
                                        <span className="w-1 h-1 bg-secondary rounded-full mr-3"></span>
                                        <span className="text-muted-foreground text-sm">{link}</span>
                                    </li>
                                ))}
                            </ul>
                        </div>

                        {/* Advantages Section */}
                        <div className="space-y-4">
                            <h3 className="text-foreground font-semibold text-lg">Üstünlüklərimiz</h3>
                            <ul className="space-y-2">
                                {advantagesList.map((advantage, index) => (
                                    <li key={index} className="flex items-center">
                                        <span className="w-1 h-1 bg-secondary rounded-full mr-3"></span>
                                        <span className="text-muted-foreground text-sm">{advantage}</span>
                                    </li>
                                ))}
                            </ul>
                        </div>
                    </div>

                    {/* Bottom Section */}
                    <div className="mt-12 pt-6 border-t border-border">
                        <div className="flex flex-col md:flex-row justify-between items-center space-y-4 md:space-y-0">
                            <div className="text-sm text-gray-400 flex flex-wrap items-center gap-1">
                                <span className="inline-flex items-center bg-secondary text-secondary-foreground px-3 py-1 rounded-md font-medium text-sm">
                                    Saytpro
                                </span>
                                <span>tərəfindən hazırlanıb ❤️</span>
                                <span className="mx-2">•</span>
                                <span>Based.az {currentYear} © Bütün hüquqlar qorunur</span>
                                <span className="mx-2">•</span>
                                <Link href="/privacy" className="hover:text-foreground transition-colors">Məxfilik Siyasəti</Link>
                                <span className="mx-2">•</span>
                                <Link href="/terms" className="hover:text-foreground transition-colors">Qaydalar</Link>
                            </div>
                        </div>
                    </div>
                </div>
            </footer>

            {/* Floating WhatsApp Button */}
            <div className="fixed bottom-6 left-6 z-50">
                <Link
                    href="https://wa.me/994775450755"
                    target="_blank"
                    rel="noopener noreferrer"
                    className="w-14 h-14 bg-green-500 hover:bg-green-600 rounded-full flex items-center justify-center shadow-lg hover:shadow-xl transition-all duration-300 group animate-pulse hover:animate-none"
                    aria-label="WhatsApp ilə əlaqə"
                >
                    <IoLogoWhatsapp className="w-7 h-7 text-white group-hover:scale-110 transition-transform" />
                </Link>
            </div>
        </>
    );
}
