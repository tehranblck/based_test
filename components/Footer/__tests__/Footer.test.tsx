import { render, screen } from "@testing-library/react";
import Footer from "../Footer";

describe("Footer", () => {
    it("renders the company name", () => {
        render(<Footer />);

        expect(screen.getByText("BASED.AZ")).toBeInTheDocument();
    });

    it("renders the company description", () => {
        render(<Footer />);

        expect(screen.getByText(/Rəqəmsal hesablar, abunəliklər, oyunlar/)).toBeInTheDocument();
    });

    it("renders contact section", () => {
        render(<Footer />);

        expect(screen.getByText("Əlaqə")).toBeInTheDocument();
        expect(screen.getByText("Əlaqə nömrəsi: +994775450755")).toBeInTheDocument();
        expect(screen.getByText("Email: contact@based.az")).toBeInTheDocument();
    });

    it("renders discover section links", () => {
        render(<Footer />);

        expect(screen.getByText("Kəşf et")).toBeInTheDocument();
        expect(screen.getByText("Hesablar və Abunəliklər")).toBeInTheDocument();
        expect(screen.getByText("Proqram Lisenziyaları")).toBeInTheDocument();
        expect(screen.getByText("Panel Xidmətləri və Alətlər")).toBeInTheDocument();
        expect(screen.getByText("Sayt və Marketing Halləri")).toBeInTheDocument();
        expect(screen.getByText("Rəqəmsal Məhsullar və Alətlər")).toBeInTheDocument();
    });

    it("renders advantages section", () => {
        render(<Footer />);

        expect(screen.getByText("Üstünlüklərimiz")).toBeInTheDocument();
        expect(screen.getByText("Sürətli Təhvil")).toBeInTheDocument();
        expect(screen.getByText("Etibarlı Xidmət")).toBeInTheDocument();
        expect(screen.getByText("24/7 Dəstək")).toBeInTheDocument();
        expect(screen.getByText("Sərfəli Qiymətlər")).toBeInTheDocument();
        expect(screen.getByText("Avtomatlaşdırılmış Sistem")).toBeInTheDocument();
    });

    it("renders social media links", () => {
        render(<Footer />);

        const socialLinks = screen.getAllByRole("link");
        const facebookLink = socialLinks.find(link => link.getAttribute("href")?.includes("facebook.com"));
        const instagramLink = socialLinks.find(link => link.getAttribute("href")?.includes("instagram.com"));
        const telegramLink = socialLinks.find(link => link.getAttribute("href")?.includes("t.me"));
        const whatsappLinks = socialLinks.filter(link => link.getAttribute("href")?.includes("wa.me"));
        const youtubeLink = socialLinks.find(link => link.getAttribute("href")?.includes("youtube.com"));

        expect(facebookLink).toBeInTheDocument();
        expect(instagramLink).toBeInTheDocument();
        expect(telegramLink).toBeInTheDocument();
        expect(whatsappLinks.length).toBeGreaterThan(0);
        expect(youtubeLink).toBeInTheDocument();
    });

    it("renders footer with dark styling", () => {
        const { container } = render(<Footer />);
        const footer = container.querySelector("footer");

        expect(footer).toHaveClass("text-white", "relative");
    });

    it("has responsive grid layout", () => {
        const { container } = render(<Footer />);
        const gridContainer = container.querySelector(".grid");

        expect(gridContainer).toHaveClass("grid-cols-1", "md:grid-cols-4", "gap-8");
    });

    it("renders section headings with yellow color", () => {
        render(<Footer />);

        const contactHeading = screen.getByText("Əlaqə");
        const discoverHeading = screen.getByText("Kəşf et");
        const advantagesHeading = screen.getByText("Üstünlüklərimiz");

        expect(contactHeading).toHaveClass("text-yellow-400", "font-semibold", "text-lg");
        expect(discoverHeading).toHaveClass("text-yellow-400", "font-semibold", "text-lg");
        expect(advantagesHeading).toHaveClass("text-yellow-400", "font-semibold", "text-lg");
    });

    it("renders copyright with current year", () => {
        render(<Footer />);
        const currentYear = new Date().getFullYear();

        expect(screen.getByText(`Based.az ${currentYear} © Bütün hüquqlar qorunur`, { exact: false })).toBeInTheDocument();
    });

    it("renders bottom section with Saytpro credit", () => {
        render(<Footer />);

        expect(screen.getByText("Saytpro")).toBeInTheDocument();
        expect(screen.getByText("tərəfindən hazırlanıb ❤️")).toBeInTheDocument();
        expect(screen.getByText("Məxfilik Siyasəti")).toBeInTheDocument();
        expect(screen.getByText("Qaydalar")).toBeInTheDocument();
    });

    it("renders Saytpro badge with purple styling", () => {
        render(<Footer />);

        const saytproBadge = screen.getByText("Saytpro");
        expect(saytproBadge).toHaveClass("bg-purple-600", "text-white", "rounded-md", "font-medium");
    });

    it("contact links have correct href attributes", () => {
        render(<Footer />);

        const phoneLink = screen.getByText("Əlaqə nömrəsi: +994775450755").closest("a");
        const emailLink = screen.getByText("Email: contact@based.az").closest("a");

        expect(phoneLink).toHaveAttribute("href", "tel:+994775450755");
        expect(emailLink).toHaveAttribute("href", "mailto:contact@based.az");
    });

    it("social links open in new tab", () => {
        render(<Footer />);

        const socialLinks = screen.getAllByRole("link").filter(link =>
            link.getAttribute("href")?.includes("facebook.com") ||
            link.getAttribute("href")?.includes("instagram.com") ||
            link.getAttribute("href")?.includes("t.me") ||
            link.getAttribute("href")?.includes("wa.me") ||
            link.getAttribute("href")?.includes("youtube.com")
        );

        socialLinks.forEach(link => {
            expect(link).toHaveAttribute("target", "_blank");
            expect(link).toHaveAttribute("rel", "noopener noreferrer");
        });
    });

    it("has proper accessibility attributes for social links", () => {
        render(<Footer />);

        const socialLinks = screen.getAllByRole("link").filter(link =>
            link.getAttribute("aria-label")
        );

        expect(socialLinks.length).toBeGreaterThanOrEqual(5);
    });

    it("renders floating WhatsApp button", () => {
        render(<Footer />);

        const whatsappButtons = screen.getAllByLabelText("WhatsApp ilə əlaqə");
        expect(whatsappButtons.length).toBeGreaterThanOrEqual(1);
        expect(whatsappButtons[0]).toHaveAttribute("href", "https://wa.me/994775450755");
    });

    it("floating WhatsApp button has correct styling", () => {
        const { container } = render(<Footer />);
        const whatsappButton = container.querySelector(".fixed.bottom-6.left-6");

        expect(whatsappButton).toBeInTheDocument();
        expect(whatsappButton?.querySelector("a")).toHaveClass("bg-green-500", "hover:bg-green-600", "animate-pulse");
    });

    it("renders bullet points for lists", () => {
        const { container } = render(<Footer />);
        const bulletPoints = container.querySelectorAll(".bg-yellow-400.rounded-full");

        expect(bulletPoints.length).toBeGreaterThan(0);
    });

    it("has hover effects on social links", () => {
        const { container } = render(<Footer />);
        const socialLinks = container.querySelectorAll(".bg-gray-700");

        expect(socialLinks.length).toBe(5);
    });

    it("matches snapshot", () => {
        const { container } = render(<Footer />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
