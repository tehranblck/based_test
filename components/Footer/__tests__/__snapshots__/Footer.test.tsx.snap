// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Footer matches snapshot 1`] = `
<footer
  class="bg-gradient-to-b from-gray-950 via-gray-900 to-black text-white relative overflow-hidden"
>
  <div
    class="pointer-events-none absolute inset-0 bg-[radial-gradient(ellipse_at_top,rgba(253,224,71,0.06),transparent_60%)]"
  />
  <div
    class="pointer-events-none absolute inset-x-0 top-0 h-px bg-gradient-to-r from-transparent via-yellow-400/20 to-transparent"
  />
  <div
    class=" max-w-[90rem] mx-auto px-4 py-12 relative z-10"
  >
    <div
      class="mb-10 rounded-xl border border-white/10 bg-white/5 p-4 sm:p-6 backdrop-blur-sm flex flex-col md:flex-row items-center justify-between gap-3"
    >
      <div
        class="text-center md:text-left"
      >
        <p
          class="text-sm text-gray-300"
        >
          Sualınız var? Dəstək komandamız 24/7 onlayndır.
        </p>
        <p
          class="text-base font-semibold text-yellow-400"
        >
          Bizə yazın, dərhal cavab alacaqsınız.
        </p>
      </div>
      <div
        class="flex items-center gap-3"
      >
        <a
          aria-label="WhatsApp ilə əlaqə"
          class="inline-flex items-center justify-center rounded-md bg-green-500 hover:bg-green-600 text-black font-medium px-4 py-2 transition-colors"
          href="https://wa.me/994775450755"
          rel="noopener noreferrer"
          target="_blank"
        >
          WhatsApp
        </a>
        <a
          aria-label="Telegram ilə əlaqə"
          class="inline-flex items-center justify-center rounded-md bg-blue-500 hover:bg-blue-600 text-white font-medium px-4 py-2 transition-colors"
          href="https://t.me/basedaz"
          rel="noopener noreferrer"
          target="_blank"
        >
          Telegram
        </a>
      </div>
    </div>
    <div
      class="grid grid-cols-1 md:grid-cols-4 gap-8"
    >
      <div
        class="space-y-4"
      >
        <div
          class="flex items-center space-x-3"
        >
          <span
            class="text-2xl font-bold text-yellow-400"
          >
            BASED.AZ
          </span>
        </div>
        <p
          class="text-gray-300 leading-relaxed text-sm"
        >
          Rəqəmsal hesablar, abunəliklər, oyunlar, proqram lisenziyaları və digər bütün ehtiyaclarınızı tək platformada rahat, sürətli və sərfəli şəkildə qarşılamaq mümkündür. Seçim edin, arxayın olun və xidmət keyfiyyətini hiss edin.
        </p>
      </div>
      <div
        class="space-y-4"
      >
        <h3
          class="text-yellow-400 font-semibold text-lg"
        >
          Əlaqə
        </h3>
        <div
          class="space-y-3"
        >
          <div>
            <a
              class="text-gray-300 hover:text-yellow-400 transition-colors duration-200 text-sm"
              href="tel:+994775450755"
            >
              Əlaqə nömrəsi: +994775450755
            </a>
          </div>
          <div>
            <a
              class="text-gray-300 hover:text-yellow-400 transition-colors duration-200 text-sm"
              href="mailto:contact@based.az"
            >
              Email: contact@based.az
            </a>
          </div>
        </div>
        <div
          class="flex space-x-3 pt-4"
        >
          <a
            aria-label="WhatsApp"
            class="w-8 h-8 bg-gray-700 hover:bg-green-500 rounded flex items-center justify-center transition-all duration-200 group"
            href="https://wa.me/994775450755"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              class="w-4 h-4 text-white"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M380.9 97.1C339 55.1 283.2 32 223.9 32c-122.4 0-222 99.6-222 222 0 39.1 10.2 77.3 29.6 111L0 480l117.7-30.9c32.4 17.7 68.9 27 106.1 27h.1c122.3 0 224.1-99.6 224.1-222 0-59.3-25.2-115-67.1-157zm-157 341.6c-33.2 0-65.7-8.9-94-25.7l-6.7-4-69.8 18.3L72 359.2l-4.4-7c-18.5-29.4-28.2-63.3-28.2-98.2 0-101.7 82.8-184.5 184.6-184.5 49.3 0 95.6 19.2 130.4 54.1 34.8 34.9 56.2 81.2 56.1 130.5 0 101.8-84.9 184.6-186.6 184.6zm101.2-138.2c-5.5-2.8-32.8-16.2-37.9-18-5.1-1.9-8.8-2.8-12.5 2.8-3.7 5.6-14.3 18-17.6 21.8-3.2 3.7-6.5 4.2-12 1.4-32.6-16.3-54-29.1-75.5-66-5.7-9.8 5.7-9.1 16.3-30.3 1.8-3.7.9-6.9-.5-9.7-1.4-2.8-12.5-30.1-17.1-41.2-4.5-10.8-9.1-9.3-12.5-9.5-3.2-.2-6.9-.2-10.6-.2-3.7 0-9.7 1.4-14.8 6.9-5.1 5.6-19.4 19-19.4 46.3 0 27.3 19.9 53.7 22.6 57.4 2.8 3.7 39.1 59.7 94.8 83.8 35.2 15.2 49 16.5 66.6 13.9 10.7-1.6 32.8-13.4 37.4-26.4 4.6-13 4.6-24.1 3.2-26.4-1.3-2.5-5-3.9-10.5-6.6z"
              />
            </svg>
          </a>
          <a
            aria-label="Instagram"
            class="w-8 h-8 bg-gray-700 hover:bg-pink-500 rounded flex items-center justify-center transition-all duration-200 group"
            href="https://instagram.com/based.az"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              class="w-4 h-4 text-white"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              />
            </svg>
          </a>
          <a
            aria-label="Telegram"
            class="w-8 h-8 bg-gray-700 hover:bg-blue-500 rounded flex items-center justify-center transition-all duration-200 group"
            href="https://t.me/basedaz"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              class="w-4 h-4 text-white"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 496 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M248 8C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm121.8 169.9l-40.7 191.8c-3 13.6-11.1 16.9-22.4 10.5l-62-45.7-29.9 28.8c-3.3 3.3-6.1 6.1-12.5 6.1l4.4-63.1 114.9-103.8c5-4.4-1.1-6.9-7.7-2.5l-142 89.4-61.2-19.1c-13.3-4.2-13.6-13.3 2.8-19.7l239.1-92.2c11.1-4 20.8 2.7 17.2 19.5z"
              />
            </svg>
          </a>
          <a
            aria-label="Facebook"
            class="w-8 h-8 bg-gray-700 hover:bg-blue-600 rounded flex items-center justify-center transition-all duration-200 group"
            href="https://facebook.com/based.az"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              class="w-4 h-4 text-white"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
              />
            </svg>
          </a>
          <a
            aria-label="YouTube"
            class="w-8 h-8 bg-gray-700 hover:bg-red-500 rounded flex items-center justify-center transition-all duration-200 group"
            href="https://youtube.com/@based.az"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              class="w-4 h-4 text-white"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 576 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
              />
            </svg>
          </a>
        </div>
      </div>
      <div
        class="space-y-4"
      >
        <h3
          class="text-yellow-400 font-semibold text-lg"
        >
          Kəşf et
        </h3>
        <ul
          class="space-y-2"
        >
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Hesablar və Abunəliklər
            </span>
          </li>
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Proqram Lisenziyaları
            </span>
          </li>
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Panel Xidmətləri və Alətlər
            </span>
          </li>
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Sayt və Marketing Halləri
            </span>
          </li>
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Rəqəmsal Məhsullar və Alətlər
            </span>
          </li>
        </ul>
      </div>
      <div
        class="space-y-4"
      >
        <h3
          class="text-yellow-400 font-semibold text-lg"
        >
          Üstünlüklərimiz
        </h3>
        <ul
          class="space-y-2"
        >
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Sürətli Təhvil
            </span>
          </li>
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Etibarlı Xidmət
            </span>
          </li>
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              24/7 Dəstək
            </span>
          </li>
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Sərfəli Qiymətlər
            </span>
          </li>
          <li
            class="flex items-center"
          >
            <span
              class="w-1 h-1 bg-yellow-400 rounded-full mr-3"
            />
            <span
              class="text-gray-300 text-sm"
            >
              Avtomatlaşdırılmış Sistem
            </span>
          </li>
        </ul>
      </div>
    </div>
    <div
      class="mt-12 pt-6 border-t border-white/10"
    >
      <div
        class="flex flex-col md:flex-row justify-between items-center space-y-4 md:space-y-0"
      >
        <div
          class="text-sm text-gray-400 flex flex-wrap items-center gap-1"
        >
          <span
            class="inline-flex items-center bg-purple-600 text-white px-3 py-1 rounded-md font-medium text-sm"
          >
            Saytpro
          </span>
          <span>
            tərəfindən hazırlanıb ❤️
          </span>
          <span
            class="mx-2"
          >
            •
          </span>
          <span>
            Based.az 
            2025
             © Bütün hüquqlar qorunur
          </span>
          <span
            class="mx-2"
          >
            •
          </span>
          <a
            class="hover:text-yellow-400 transition-colors"
            href="/privacy"
          >
            Məxfilik Siyasəti
          </a>
          <span
            class="mx-2"
          >
            •
          </span>
          <a
            class="hover:text-yellow-400 transition-colors"
            href="/terms"
          >
            Qaydalar
          </a>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
